CONDUCT.md

```diff
@@ -0,0 +1,75 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Do you want this to read "Clawpack Community Code of Conduct" or similar? It is, after all, a code of conduct meant for members of the Clawpack community, not for the Contributor Covenant project. Compare:

https://golang.org/conduct
https://swift.org/community/#code-of-conduct
https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html